README.md

```diff
@@ -475,6 +502,16 @@ You can subscribe on the following `$SYS` topics to get client presence:
  - `$SYS/+/disconnect/clients` - will inform about client disconnections.
 The payload will contain the `clientId` of the connected/disconnected client
 
+## Plugins
```

Comment:
```suggestion
## Persistences
```

Reply:
@mcollina I think plugins is better as stats and logs are not persistences